pkgs/applications/networking/syncplay/trusted_certificates.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/syncplay/client.py b/syncplay/client.py
+index b7cb245..be72d94 100755
+--- a/syncplay/client.py
++++ b/syncplay/client.py
+@@ -848,6 +848,7 @@ class SyncplayClient(object):
+         self._endpoint = HostnameEndpoint(reactor, host, port)
+         try:
+             certs = pem.parse_file(SSL_CERT_FILE)
++            certs = [cert for cert in certs if type(cert) is pem.Certificate]
```

Comment:
Using `isinstance` here would probably be more idiomatic.

Reply:
doesn't work here, since the bad kind is a subclass of the good kind (and imo it's cleaner to allowlist rather than exclude the bad kind specifically)